logrhythmnextgensiem.json

```diff
@@ -0,0 +1,987 @@
+                        "",
+                        "New",
+                        "Open",
+                        "Closed"
+                    ],
+                    "order": 1
+                },
+                "rbp": {
+                    "description": "New RBP for the alarm",
+                    "data_type": "numeric",
+                    "order": 2
+                },
+                "comment": {
+                    "description": "New comment to add to alarm",
+                    "data_type": "string",
+                    "order": 3
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.comment",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.parameter.id",
+                    "data_type": "string",
+                    "contains": [
+                        "logrhythm alarm id"
+                    ],
+                    "column_name": "Alarm ID",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.parameter.rbp",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.parameter.status",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.responseMessage",
+                    "data_type": "string",
+                    "column_name": "Message",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 1,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get events",
+            "identifier": "get_events",
```

Comment:
nit: Since we're returning a list of events here, should we call this `list events`? (see [reference](https://github.com/splunk-soar-connectors/carbonblackresponse/blob/next/carbonblack.json#L7253))

Reply:
This should be 'get events' as we fetching events of specific Alarm ID. When we are filtering based on some params and fetching events, that time it should be 'list events'.

In this case, I feel that we should keep 'get events'.